critiquebrainz/frontend/external/musicbrainz_db/entities.py

```diff
@@ -0,0 +1,46 @@
+from critiquebrainz.frontend.external.musicbrainz_db.release_group import fetch_multiple_release_groups, get_release_group_by_id
+from critiquebrainz.frontend.external.musicbrainz_db.place import fetch_multiple_places, get_place_by_id
+from critiquebrainz.frontend.external.musicbrainz_db.event import fetch_multiple_events, get_event_by_id
+
+
+def get_multiple_entities(entities):
+    """Fetch multiple entities using their mbids.
+
+    Args:
+        entites: List of tuples containing the entity_id and the entity_type.
+
+    Returns:
+        Dictionary containing the entities keyed by their mbid.
+    """
+    entities_info = {}
+    release_group_mbids = [entity[0] for entity in filter(lambda entity: entity[1] == 'release_group', entities)]
+    place_mbids = [entity[0] for entity in filter(lambda entity: entity[1] == 'place', entities)]
+    event_mbids = [entity[0] for entity in filter(lambda entity: entity[1] == 'event', entities)]
+    if release_group_mbids:
```

Comment:
This and other checks below seem redundant. If an empty list is passed we expect an empty output anyway, right?